distribution/lib/Standard/Base/0.0.0-dev/src/Polyglot/Proxy_Polyglot_Array.enso

```diff
@@ -0,0 +1,23 @@
+from Standard.Base import Polyglot
+
+## Advanced
+
+    Wrapper for Polyglot Arrays
```

Comment:
I don't understand why there should be a specific, user visible type for polyglot arrays. There shall be (at most) only single`Array` and that one has to work with all polyglot arrays. There are no other arrays anywhere!

Reply:
if we can make it so polyglot arrays and Enso arrays are just the same this would be great - however I'd like to separate that from this tidy up as this will allow us to use the Vector type for a few extra cases we need.

I will confirm with Sylwia and add a ticket to make one array to rule them all.